src/main/java/net/imagej/legacy/plugin/MacroAutoCompletionProvider.java

```diff
@@ -173,9 +176,12 @@ public void addMacroExtensionAutoCompletions(MacroExtensionAutoCompletionService
 		this.macroExtensionAutoCompletionService = macroExtensionAutoCompletionService;
 
 		List<BasicCompletion> completions = macroExtensionAutoCompletionService.getCompletions(this);
+		List<Completion> completionsCopy = new ArrayList<Completion>();
```

Comment:
Remove redundant type arguments:
```suggestion
		List<Completion> completionsCopy = new ArrayList<>();
```